docs/2.0. EDID to Receiver Caps mapping.md

```diff
@@ -1,84 +1,192 @@
+    }
+  },
+  {
+    "urn:x-nmos:cap:format:frame_width": {
+      "enum": [ 1024 ]
+    },
+    "urn:x-nmos:cap:format:frame_height": {
+      "enum": [ 768 ]
+    },
+    "urn:x-nmos:cap:format:interlace_mode": {
+      "enum": [ "progressive" ]
+    },
+    "urn:x-nmos:cap:format:grain_rate": {
+      "enum": [ { "numerator": 60 } ]
+    }
+  }
+]
+```
+
+</details>
 
-Each video timing MUST be present in Receiver Capabilities as a Constraint Set with non-empty `urn:x-nmos:cap:format:frame_width`, `urn:x-nmos:cap:format:frame_height` and `urn:x-nmos:cap:format:grain_rate` fields. There are multiple E-EDID blocks which keep information about timings each with their own mapping requirements.
+#### Standard Timings
 
-#### Established Timings
+Standard Timing format is defined in [E-EDID][E-EDID] section 3.9. The mapping is as follows:
 
-There are three sets of timings of this type: Established Timings I, II & III. The first two types are defined in [E-EDID][E-EDID] section 3.8, the last - in [E-EDID][E-EDID] section 3.10.3.9.
+* `urn:x-nmos:cap:format:frame_width` MUST be calculated from the _Horizontal Active Pixel Count_
+* `urn:x-nmos:cap:format:frame_height` MUST be calculated using the _Frame Width_ and _Image Aspect Ratio_
+* `urn:x-nmos:cap:format:grain_rate` MUST be calculated with _Field Refresh Rate_
 
-Each Established Timing is an ID associated with a union of frame width, height and rate and interlace mode, and the mentioned sections define this mapping.
+<details><summary>Example: Standard Timings</summary>
 
-#### Standard Timings
+For example, given the bytes of Standard Timings were `D1 C0 01 01 01 01 01 01 01 01 01 01 01` then the receiver capabilies could contain the following in its constraint sets
 
-Standard Timing format is defined in [E-EDID][E-EDID] section 3.9.
+```json
+[
+  {
+    "urn:x-nmos:cap:format:frame_width": {
+      "enum": [ 1080 ]
+    },
+    "urn:x-nmos:cap:format:frame_height": {
+      "enum": [ 1920 ]
+    },
+    "urn:x-nmos:cap:format:grain_rate": {
+      "enum": [ { "numerator": 60 } ]
+    }
+  }
+]
+```
 
-Frame width MUST be calculated using the following formula: Frame width = (1B + 31) × 8, where 1B is the 1st byte of a Standard Timing. It MUST go into `urn:x-nmos:cap:format:frame_width`.
+</details>
 
-Frame height MUST be calculated using the following formula: Frame height = FW / IAR, where FW is Frame width and IAR is _Image Aspect Ratio_, and go into `urn:x-nmos:cap:format:frame_height`.
+#### Detailed Timing Descriptors (18 Byte Descriptors)
 
-Frame rate is named _Field Refresh Rate_ there and MUST go into `urn:x-nmos:cap:format:grain_rate`.
+Defined in [E-EDID][E-EDID] section 3.10, there a 4 possible descriptors which can be provided. The first, _Prefered Timing Mode_ is an obligation and MUST have a `urn:x-nmos:cap:meta:preference` value of `100`. Any additional Detailed Timing Descriptors in [CTA-861][CTA-861] Extension BLock (section 7.2.1) MUST follow the mapping.
```

Comment:
```suggestion
Defined in [E-EDID][E-EDID] section 3.10, there are 4 possible descriptors which can be provided. The first, _Prefered Timing Mode_ is an obligation and MUST have a `urn:x-nmos:cap:meta:preference` value of `100`. Any additional Detailed Timing Descriptors in [CTA-861][CTA-861] Extension BLock (section 7.2.1) MUST follow the mapping.
```
What does "follow the mapping" mean?

Reply:
I stole the words from the next paragraph

![image](https://user-images.githubusercontent.com/16867443/114758548-15cf6500-9d2b-11eb-9950-ed2168f10f51.png)

Reply:
This sentence makes me feeling like _Preferred Timing Mode_ should follow some other mapping because of "additional". Do you mind removing this word?